go.mod

```diff
@@ -14,9 +15,14 @@ require (
 	k8s.io/utils v0.0.0-20240102154912-e7106e64919e
 	sigs.k8s.io/controller-runtime v0.16.3
 	sigs.k8s.io/controller-tools v0.14.0
+	sigs.k8s.io/gateway-api/conformance/grpc-echo v0.0.0-00010101000000-000000000000
 	sigs.k8s.io/yaml v1.4.0
 )
 
+// Needed because of the go.sum and go.mod used for Dockerfile
+// The same generated code is reused in the conformance tests.
+replace sigs.k8s.io/gateway-api/conformance/grpc-echo => ./conformance/grpc-echo
+
```

Comment:
just curious if this change is temporary only as it might fail go import from other project/module.

Reply:
Good catch. I hadn't noticed this sharp edge. I've corrected this.